base/applications/mscutils/eventvwr/lang/ro-RO.rc

```diff
@@ -229,12 +229,12 @@ ID eveniment:        %s\r\n\
 Dată:                %s\r\n\
 Oră:                 %s\r\n\
 Utilizator:          %s\r\n\
-Calculator:          %s\r\n\
+Computer:            %s\r\n\
```

Comment:
E.g. Here you call it Computer. Which I do think is the better translation indeed.